extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -515,6 +519,8 @@ export class AmpSlideScroll extends BaseSlides {
           setStyle(this.slideWrappers_[i], 'order', '');
         }
         this.slideWrappers_[i].classList.remove(SHOWN_CSS_CLASS);
+        this.slides_[i] &&
+          this.slides_[i].setAttribute('aria-hidden', 'true');
```

Comment:
@camelburrito We need this here otherwise we can get to a conflictin state where `display:none` but `aria-hidden=false` (this can happens if one jumps more than 2 slides, although not something that UI supports yet, it is something that can happen when we do amp-bind)